<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pdb_res.qrc">
    <normaloff>:/images/images/chest.png</normaloff>:/images/images/chest.png</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MyTree" name="m_TreeOfNodes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>93</y>
      <width>256</width>
      <height>241</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::MoveAction</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="m_DBNameList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>30</height>
     </size>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="MyEditor" name="m_textEditor">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>44</y>
      <width>256</width>
      <height>141</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>120</height>
     </size>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="m_EditorLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>Node descriptor</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_AttachmentLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>Attachments</string>
    </property>
   </widget>
   <widget class="MyTable" name="m_Service_Tab">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>120</height>
     </size>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragOnly</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::MoveAction</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textDiagnostic">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>104</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect_to_DB">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionCreate_new">
   <property name="text">
    <string>Create new</string>
   </property>
  </action>
  <action name="actionCreate_new_2">
   <property name="text">
    <string>Create new...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyTree</class>
   <extends>QTreeWidget</extends>
   <header>mytree.h</header>
  </customwidget>
  <customwidget>
   <class>MyTable</class>
   <extends>QTableWidget</extends>
   <header>mytable.h</header>
  </customwidget>
  <customwidget>
   <class>MyEditor</class>
   <extends>QTextEdit</extends>
   <header>myeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="pdb_res.qrc"/>
 </resources>
 <connections/>
</ui>
